python/tk_multi_workfiles/ui/file_group_widget.py

```diff
@@ -106,7 +106,7 @@ def setupUi(self, FileGroupWidget):
 "font: 11px;\n"
 "color: grey;\n"
 "}")
-        self.msg_label.setAlignment(QtCore.Qt.AlignLeading|QtCore.Qt.AlignLeft|QtCore.Qt.AlignVCenter)
+        self.msg_label.setAlignment(QtCore.Qt.AlignLeading|QtCore.Qt.AlignLeft|QtCore.Qt.AlignTop)
```

Comment:
Text was centered and it looked really bad when some messages wrapped on multiple lines and some didn't.